src/tests/unit/tests/background/scanner-utility.test.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { It, Mock, MockBehavior } from 'typemoq';
+import { isFunction } from 'lodash';
```

Comment:
Build failing with ERROR: /Users/vsts/agent/2.149.2/work/1/s/src/tests/unit/tests/background/scanner-utility.test.ts:4:1 - Import sources within a group must be alphabetized.

Reply:
My bad... I was doing this manually... I guess I miss this two files.